Snippets/Core/Core_7/Headers/OverrideConversationId.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+using Core7.Handlers;
+using NServiceBus;
+
+public class ConversationIdHeaderOverride
```

Comment:
@ramonsmits I'm about to add doco for https://github.com/Particular/NServiceBus/blob/74a1c4bddfc5167deeb8fcd010fe0644f25823fa/src/NServiceBus.AcceptanceTests/Core/Causation/When_overriding_conversation_id_generation.cs#L53 

I wonder if we should make that the only documented option for v7? (or is there still a usecase for "poluting" the business code with setting the header)

Reply:
To be honest, I think the business code is the perfect place to indicate what the conversation ID is. I personally would want this magic per message to be hidden in a mutator or pipeline unless it can be really applied generically.

Reply:
For example, you could use a convention based on message ID, take the first part of a message ID to identify the conversation: